### ISSUE TITLE
unable to update the language

### ISSUE THREAD

**Issue by galamarco** [joomla/joomla-cms#35512]
### Steps to reproduce the issue
I have a clean installation of Joomla 4.0.2 with Italian language installed for frontend and backend. 
The update system notifies me a language update available. I run this update, it is not carried out and after a few hours an update is available again the same update that I run, it tells me that it has completed the update but it is not so and still in loop.
I think Joomla fails to update the language.
I don't know if it has anything to do with this problem but the current language version is not detected.
![Current version not detected](https://bramantehouse.it/images/articoli/2021-09/screenshot_29.png?1631174095435)

### Expected result
If I do an update it must actually be done.

### Actual result
The update does not happen but it confirms that I have done it correctly.

### System information (as much as possible)
Joomla 4.0.2
PHP: 7.4.23
MySql: 10.3.31-MariaDB-cll-lve

Marco Galassi

**Comment by brianteeman** [comment]
The problem is because the italian language xml file does not contain version information.

@HLeithner can you assist?
```

<?xml version="1.0" encoding="utf-8"?>
<extension client="administrator" type="language" method="upgrade">
	<name>Italiano (it-IT)</name>
	<tag>it-IT</tag>
	<version/>
	<creationDate>August 2021</creationDate>
	<author>Italian Translation Team</author>
	<authorEmail>stellainformatica@gmail.com</authorEmail>
	<authorUrl>www.joomla.org</authorUrl>
	<copyright>(C) 2013 Open Source Matters, Inc.</copyright>
	<license>GNU General Public License version 2 or later; see LICENSE.txt</license>
	<description>it-IT administration language</description>
	<files>
		<folder>/</folder>
		<filename file="meta">install.xml</filename>
	</files>
	<params/>
</extension>

```

**Comment by infograf768** [comment]
The version is absent from ALL xml(s). Again one of these Crowdin oddities not corrected by the TT.

**Comment by brianteeman** [comment]
> The version is absent from ALL xml(s). Again one of these Crowdin oddities not corrected by the TT.

Of the 7 I just checked it was only missing from 2

**Comment by HLeithner** [comment]
hmm I checked the it package in https://github.com/joomla/core-translations/blob/main/joomla_v4/translations/package/it-IT/language/it-IT/langmetadata.xml and this is correct maybe 4.02 packages are broken?

**Comment by brianteeman** [comment]
The latest package available to download by a user is https://downloads.joomla.org/language-packs/translations-joomla4/downloads/joomla4-italian/4-0-2-3

This does not have the version information

**Comment by conconnl** [comment]
> The version is absent from ALL xml(s). Again one of these Crowdin oddities not corrected by the TT.

This has nothing to do with Crowdin. 
Has everything to do with a TT not making sure everything is checked, approved and tested. 

If they don’t update or add the version number which is required by how joomla works, you get issues.

**Comment by infograf768** [comment]
@HLeithner 
The Italian package has been updated after I contacted the TT and the pkg_it-IT.xml version has been corrected i.e. 4.0.2.4 as pack zip name.
But what you quote, which has been modified at the same time, is wrong as the pack is for 4.0.2 and not 4.0.3...
This comes from the fact that the version has changed when 4.0-dev has been reset to dev after 4.0.2 release. Crowdin is therefore proposing 4.0.3 and it's easy to mistake.  Happily, the site, admin and api versions in their xml have no impact on updating, afaik only the package.xml counts.

@conconnl 
You are partly right. Crowdin is not only at stake. Responsibility is also to Marco and others with forcing the new system and the absence of coordination checking packs.

**Comment by infograf768** [comment]
@conconnl 
As I don't know where to post this, here is one of my findings on Crowdin:

<img width="1034" alt="Screen Shot 2021-09-10 at 09 31 03" src="https://user-images.githubusercontent.com/869724/132817247-0f684351-64de-4d03-9762-af4f115a4d65.png">

This confuses `Pashto` tag `ps-AF` with `Afrikaans` => tag `af-ZA`,
`Dari Persian` tag `prs-AF` for which we had a pack in J3 is absent from the list.

**Comment by brianteeman** [comment]
PLEASE CREATE YOUR OWN ISSUES AND NOT HIJACKING OTHERS

**Comment by conconnl** [comment]
> PLEASE CREATE YOUR OWN ISSUES AND NOT HIJACKING OTHERS

Exactly, you know there is also a Repository for the translation storage.
https://github.com/joomla/Core-Translations

**Comment by HLeithner** [comment]
> > PLEASE CREATE YOUR OWN ISSUES AND NOT HIJACKING OTHERS
> 
> Exactly, you know there is also a Repository for the translation storage.
> https://github.com/joomla/Core-Translations

I'm moving this issue to core translations (not sure if this is really the right issue tracker because it's a translation italien issue) at least it has nothing todo with the core it self.
